site/_quarto-exe-demo.yml

```diff
@@ -0,0 +1,12 @@
+format:
```

Comment:
Maybe not for this PR — because this is already like watching _Gojira_ unleash its full power on the unsuspecting populace — but it might make sense to change all the “demo” stuff to be “development”, to match the Heap stuff which in turn would match what the code does. 

All the demo stuff is madey-uppy and goes back to Andres and I riffing on the prehistoric proto docs site. This also explains why the [PR previews use the `development profile](https://github.com/validmind/documentation/blob/main/.github/workflows/validate-docs-site.yaml#L35). At some point, I hope to phase our all references to "demo".

Reply:
> but it might make sense to change all the “demo” stuff to be “development”, to match the Heap stuff which in turn would match what the code does.

Hm, yes... I did confuse myself a few times doing the Heap bits because of that discrepancy. The problem is our convention for the platform is also... not this, since "development" on that end means production!